scripts/survival_migration/survivalDataMigration.py

```diff
@@ -0,0 +1,196 @@
+import csv
+import pandas as pd
+import numpy as np
+import os
+import sys
+import argparse
+import shutil
+
+PARENT_PARSER_NAME = 'survival data migration tool'
+NULL_VALUES = ["[not applicable]", "[not available]", "[pending]", "[discrepancy]", "[completed]", "[null]", "", "na"]
+MAPPING_TO_ONE_COLUMN = "MAPPING_TO_ONE"
+MAPPING_TO_ZERO_COLUMN = "MAPPING_TO_ZERO"
+
+def migrateAttibuteValue(value, vocabulariesMappingToOne, vocabulariesMappingToZero):
+    if value.lower() in NULL_VALUES:
+        return value
+    if value.lower() in vocabulariesMappingToOne:
+        return "1:" + value
+    elif value.lower() in vocabulariesMappingToZero:
+        return "0:" + value
+    else:
+        sys.exit("cannot find the mapping for vocabulary {} , please provide the the mapping rules with option -a".format(value))
+        return value
+
+def generateVocabularies(vocabulariesMappingToOne, vocabulariesMappingToZero, vocabulariesFile):
+    df = pd.read_csv(vocabulariesFile, sep='\t')
+    oneValues = df[MAPPING_TO_ONE_COLUMN].tolist()
+    for unparsedValue in oneValues:
+        for parsedValue in unparsedValue.split(","):
```

Comment:
Thanks for the tool @dippindots! Here are a couple of issues that came up when testing it on files. If one additional vocabulary is passed, say for example `1:LIVE` to OS_STATUS, it is added to survivalStatusVocabularies.txt file as new line `live\tnan`. Running the script the second time, throws an AttributeError on split() for `nan`. Check if value exists before splitting? Also, at line 33.